pkg/reconciler/channel/channel_test.go

```diff
@@ -110,24 +111,24 @@ func TestReconcile(t *testing.T) {
 		Name: "Successful reconciliation",
 		Key:  testKey,
 		Objects: []runtime.Object{
-			NewChannel(channelName, testNS,
-				WithChannelTemplate(channelCRD()),
-				WithInitChannelConditions),
-			NewInMemoryChannel(channelName, testNS,
-				WithInitInMemoryChannelConditions,
-				WithInMemoryChannelDeploymentReady(),
-				WithInMemoryChannelServiceReady(),
-				WithInMemoryChannelEndpointsReady(),
-				WithInMemoryChannelChannelServiceReady(),
-				WithInMemoryChannelAddress(backingChannelHostname)),
+			NewChannelV1Beta1(channelName, testNS,
+				WithChannelTemplateV1Beta1(channelCRD()),
+				WithInitChannelConditionsV1Beta1),
+			NewInMemoryChannelV1Beta1(channelName, testNS,
```

Comment:
yikes we need a better way... this is out of hand 🤣

Reply:
Yeah, we could introduce reconciler/testing/v* directories and put them there? Other than that nothing comes to mind off the top of my head... Thoughts?